### DISCUSSION TITLE
No new version for MacOS

### DISCUSSION THREAD

**Discussion by zsalab** [utmapp/UTM#4012] [UTM Help (macOS)] [answered]
**Describe the issue**
The UTM.dmg is missing (mentioned in the release note) from the assets on the Github releases page, and the latest version in the App Store is 3.1.5 (yes I bought it to support the project, would be nice to update from there)

**Configuration**
* UTM Version: 
* OS Version: 
* Intel or Apple Silicon?

**Crash log**

**Debug log**

**Upload VM**

**Comment by osy** [comment, marked as answer]
Be patient young one. https://github.com/utmapp/UTM/actions/runs/2317581962
